ledfx/api/integration_qlc.py

```diff
@@ -37,10 +37,9 @@ async def get(self, integration_id) -> web.Response:
 
         response = {}
 
-        # generate dict of {effect_id: effect_name}
-        effect_names = []
+        effect_ids = []
         for effect_type, effect in self._ledfx.effects.classes().items():
-            effect_names.append(effect.NAME)
```

Comment:
are you testing on this interface?

I appreciate the shift is subtle and looks more "correct", effect_type is equivalent to id here ( I had to go look at the code ) , and id is more definitive than name.

I just worry about side effects,